codegen/templates/keyword.j2

```diff
@@ -39,6 +40,7 @@ from ansys.dyna.core.keywords.keyword_classes.auto.{{link.modulename}} import {{
 {% if keyword_data.card_sets %}
 {% for card_set in keyword_data.card_sets.sets %}
 class {{card_set.name}}(Cards):
+    """DYNA {{card_set.title}} keyword."""
```

Comment:
This isn't a keyword, it is a card set. This should be

"""{{card_set.title}} CardSet."""